deploy.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION=`date "+%Y%m%dt%H%M%S"`
+LOGFILE="$VERSION.log"
+
+MIGRATE="--migrate"
+if [ "$GCLOUD_PROJECT" = "dartlang-pub-dev" ]; then
+  MIGRATE=""
+fi
+
+ANALYZER_HOST="https://""$VERSION""-dot-analyzer-dot-""$GCLOUD_PROJECT"".appspot.com"
+SEARCH_HOST="https://""$VERSION""-dot-search-dot-""$GCLOUD_PROJECT"".appspot.com"
+APP_HOST="https://""$VERSION""-dot-""$GCLOUD_PROJECT"".appspot.com"
+
+if [ "$1" = "all" ] || [ "$1" = "analyzer" ]; then
+  OLD_ANALYZER=`gcloud app versions list --service analyzer --format='value(id)'`
+
+  echo "Deploying analyzer ($VERSION)..."
+  echo "Deploying analyzer ($VERSION)..." >>$LOGFILE
+  gcloud app deploy analyzer.yaml --no-promote -v $VERSION -q >>$LOGFILE 2>&1
+
+  echo "Checking analyzer status on $ANALYZER_HOST ..."
+  echo "Checking analyzer status on $ANALYZER_HOST ..." >>$LOGFILE
+  curl "$ANALYZER_HOST""/debug" -s -f  >/dev/null
```

Comment:
Doesn't it usually take some time until the service is actually ready? iirc it usually takes a bit more than just waiting for `gcloud app deploy` to complete.